accounting/localizations/argentina.rst

```diff
@@ -0,0 +1,280 @@
+========
+Argentina
+========
+
+Introduction
+~~~~~~~~~~~~
+The Argentinean localization has been improved and extended in Odoo V13, in this version the next modules are available:
+
+#. **l10n_ar**:  This module add accounting features for the Argentinian localization, which represent the minimal configuration needed for a company  to operate in Argentina and under the AFIP (Administración Federal de Ingresos Públicos) regulations and guidelines.
+
+#. **l10n_ar_reports**: Add VAT Book report which is a legal requirement in Argentine and that holds the VAT detail info of sales or purchases recorded on the journal entries. This module includes as well the  VAT summary report that is used to analyze the invoice
+
+
+Configuration
+~~~~~~~~~~~~~
+
+1. Install the Argentinean localization modules
+---------------------------------------------
+
+For this, go to Apps and search for Argentina. Then click Install for the first two modules.
+
+.. image:: media/argentina01.png
+   :align: center
+
+1.1 Configure your company
+-----------------------------------------------------
+
+Once that the modules are installed, the first step is to set up your company data.  Additional to  the basic information, a key field to fill in the AFIP Responsibility Type, that represent the fiscal obligation and structure of the company:
+
+.. image:: media/argentina02.png
```

Comment:
please use a tool like [pngquant](https://pngquant.org/) to compress your images. For instance, on this image, you go from 85k to 38k